extern/region-4.14/configure

```diff
@@ -4304,8 +4284,8 @@ esac
 
 
 
-macro_version='2.4.6'
-macro_revision='2.4.6'
+macro_version='2.4.2'
+macro_revision='1.3337'
```

Comment:
From https://github.com/sherpa/sherpa/pull/1204/files?file-filters%5B%5D=.ac&file-filters%5B%5D=.am&file-filters%5B%5D=.c&file-filters%5B%5D=.h&file-filters%5B%5D=.in&file-filters%5B%5D=.l&file-filters%5B%5D=.m4&file-filters%5B%5D=.org&file-filters%5B%5D=.sh&file-filters%5B%5D=.y&file-filters%5B%5D=No+extension#diff-f22710e4eb0fbe749656c275a5f14cc05805cf6c90afa013f0216809f80f7586 it looks like we've regressed the "ltmain" version from 2,4,6 to 2.4.2

From what I can tell it's not a problem as long as we include the libtool.m4 files (and probably others)  of the correct version (which we do) but looking at the details of some of these files they deal with OS system versions which makes me wonder if we would want an update of these for the Apple M1 platform (or is that not needed)?

Reply:
I also wonder if it's an issue for aarch64 support #1186 - which is not a high priority for us but the community may appreciate it?